README.md

```diff
@@ -72,116 +71,22 @@ for `DbRoleSlice` for details).
   tenant-specific roles (_TENANT_WRITER_, _TENANT_READER_) will be able to
   access only their own tenant's data.
 - **Versioning**. If a record persisted in data store has a field named
-  __revision_, versioning (revisioning) will be supported on that table. Among
+  _revision_, versioning (revisioning) will be supported on that table. Among
   multiple concurrent updates, only one will succeed.
-
-Work in progress:
-
-- **Metadata support**
+- **Metadata support** like CreatedAt, UpdatedAt, DeletedAt (using gorm.Model)
```

Comment:
Looks like these bullet points repeat the bullet points in the beginning of the **Features** section. Add **Versioning** and **Metadata support** to the bullet points above?

Reply:
done